Sources/Rendering/Core/InteractorStyle/index.d.ts

```diff
@@ -210,6 +210,19 @@ export interface vtkInteractorStyle extends vtkInteractorObserver {
    * Handles a keypress.
    */
   handleKeyPress(callData: unknown): void;
+
+  /**
+   * Explicitly defines a renderer to be used for event handling.
+   * If not set, the pokedRenderer of the event will be used.
+   *
+   * @param {vtkRenderer} renderer
+   */
+  setForcedRenderer(renderer: vtkRenderer): void;
```

Comment:
What about calling it "focusedRenderer"? I don't have a strong naming pref, but "forcedRenderer" seems a little opaque to me.

In addition, we should allow setting null here to be able to turn off this functionality.

Reply:
I agree. `focusedRenderer` sounds clearer

Reply:
the choice to use `forcedRenderer` comes from `vtkRenderWindowInteractor`  [here](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/RenderWindowInteractor/index.js#L1190)